lib/hyrax/publisher.rb

```diff
@@ -119,6 +119,10 @@ class Publisher
     #     unique id), AND a `:user` (the ::User responsible for the update).
     register_event('collection.membership.updated')
 
+    # @since 3.5.0
```

Comment:
i plan to backport this quickly to the 3.x stable branch, so this is the appropriate release target